### PR TITLE
sets og::tags and favicon for foundry book 

### PR DESCRIPTION
**Pull request by tkumor3**
Zrzut ekranu 2023-11-13 o 09 05 05

Closes #

## Introduced changes

-

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/book.toml [hunk 1]
```diff
@@ -4,6 +4,7 @@ language = "en"
 multilingual = false
 src = "src"
 title = "The Starknet Foundry Book"
+description = "Starknet Foundry is a toolchain for developing Starknet smart contracts. It helps with writing, deploying, and testing your smart contracts. It is inspired by Foundry."
```
Comment (cptartur) [inline docs/book.toml:7]:
```suggestion
description = "Starknet Foundry is a toolchain for developing Starknet smart contracts. It helps with writing, deploying, and testing your smart contracts written in Cairo. It is inspired by Foundry."
```

What about adding that?

File: docs/theme/head.hbs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+    <meta httpEquiv="Content-Language" content="en-US">
+    <meta name="apple-mobile-web-app-title" content="Starknet Foundry Book">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@swmansionxyz">
+    <meta name="twitter:creator" content="@jajakobyly">
```
Comment (cptartur) [inline docs/theme/head.hbs:5]: Can we skip this tag?

File: docs/theme/head.hbs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <meta property="og:title" content="The Starknet Foundry Book">
+    <meta property="og:description" content="{{ chapter_title }}">
```
Comment (cptartur) [inline docs/theme/head.hbs:7]: Where is `chapter_title` defined?

File: docs/theme/head.hbs [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="https://foundry-rs.github.io/starknet-foundry/images/foundry.png">
+    <meta property="og:image:alt" content="Starknet Foundry toolkit to developing Starknet contracts">
```
Comment (cptartur) [inline docs/theme/head.hbs:10]:
```suggestion
    <meta property="og:image:alt" content="Starknet Foundry toolkit for developing Starknet smart contracts">
```

File: docs/theme/head.hbs (hunk 2 above)
Comment (tkumor3) [reply docs/theme/head.hbs:5]: I have to research, because it's only copied from scarb

File: docs/theme/head.hbs (hunk 3 above)
Comment (tkumor3) [reply docs/theme/head.hbs:7]: Zrzut ekranu 2023-11-13 o 10 27 08

These are examples of `chapter_title`. In the example, I added in the description section, there is how it looks for `Fork Testing`

**Review by cptartur — APPROVED** [review]

**Pull request merged**